fuel_test/prep_env.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -x
+
+function restore_fm {
+	#Restore snapshot that has test machine's ssh public key
+	local xs_host="$1"
+	local fm_name="$2"
+	local fm_snapshot="$3"
+	ssh -qo StrictHostKeyChecking=no root@$xs_host \
+	'
+	set -x
+	#echo "FM_NAME : '$fm_name'"
+	#echo "FM_SNAPSHOT : '$fm_snapshot'"
+	vm_uuid=$(xe vm-list name-label="'$fm_name'" --minimal)
+	#echo "VM_UUID : $vm_uuid"
+	snapshot_uuid=$(xe snapshot-list name-label="'$fm_snapshot'" snapshot-of="$vm_uuid" --minimal)
+	#echo "SNAPSHOT_UUID: $snapshot_uuid"
+	xe snapshot-revert snapshot-uuid="$snapshot_uuid"
+	xe vm-start vm="'$fm_name'"
+	'
+}
+
+function create_node {
+	# Create a node server(compute/controller node) with given memory, disk and NICs
+	local xs_host="$1"
+	local vm="$2"
+	local mem="$3"
+	local disk="$4"
+	local eths=()
+	for ((i=5;i<=$#;i++)); do
```

Comment:
Probably better to have a separate 'create_node' and 'add_network' method rather than use bash magic